package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "electron",
+  "version": "v0.35.0",
```

Comment:
Current version is 0.35.4. Remove "v" to be consistent with `process.versions['electron']` and electron-prebuilt.